misc/changelog-generator/test-changelog-generator

```diff
@@ -350,6 +360,9 @@ cat > expected.txt <<EOF
 	- Complete Fix for changelog N4
 	  Should be included N5
 	- Fix for changelog N1
+	- Make sure the bugtracker reference is taken from the title
+	             and goes after this message. The title should not be
+	             included. N68 (CFE-1234)
```

Comment:
I think it shouldn't. Many entries use spaces deliberately to format the content. Your iterator fix is one of them.